app/views/categories/show.html.erb

```diff
@@ -28,7 +28,11 @@
 <% else %>
   <% category.children.each do |child| %>
     <div class="category-name">
-      <%= link_to child.name + " (#{child.children.count + child.notes.count + child.photos.count} items)", child %>
+      <% if child.children.empty? && category.notes.empty? && category.photos.empty? %>
```

Comment:
This is a lot of calls to `.empty?`. What are we looking for, that none of those things exist?

`[child.children, category.notes, category.photos].all?(&:empty?)` will also do the same thing you're doing here.